plugins/inputs/statsd/statsd_test.go

```diff
@@ -1770,3 +1770,55 @@ func TestParseNoSanitize(t *testing.T) {
 		require.Equalf(t, name, test.outName, "Expected: %s, got %s", test.outName, name)
 	}
 }
+
+func TestParse_InvalidAndRecover(t *testing.T) {
```

Comment:
```suggestion
func TestParse_InvalidAndRecoverIntegration(t *testing.T) {
```
Should this only be run nightly during integration tests?

Reply:
:facepalm: yes, good catch